mirrormanager2/health_checks.py

```diff
@@ -0,0 +1,15 @@
+from flask_healthz import HealthError
+from sqlalchemy.exc import OperationalError
+
+from mirrormanager2.database import DB
+
+
+def liveness():
+    pass
+
+
+def readiness():
+    try:
+        DB.manager.get_status()
+    except OperationalError as err:
```

Comment:
This looks good, but we can do even better! The sqlalchemy-helpers doc has an example implementation of a health check: https://sqlalchemy-helpers.readthedocs.io/en/latest/user.html#openshift-health-checks

Reply:
Ah. I'm unsure how I missed that, but PR has been updated. I also realized I could simplify the tests.

Thanks